src/layman/layer/rest_workspace_test.py

```diff
@@ -55,35 +55,35 @@
 
 METADATA_PROPERTIES_EQUAL = METADATA_PROPERTIES
 
-min_geojson = """
+MIN_GEOJSON = """
 {
   "type": "Feature",
   "geometry": null,
   "properties": null
 }
 """
 
-num_layers_before_test = 0
+NUM_LAYERS_BEFORE_TEST = 0
```

Comment:
As it is variable and not constant, I prefer lower case and `# pylint: disable=invalid-name`